contracts/strategy/liquity/LiquityStrategy.sol

```diff
@@ -246,8 +282,9 @@ contract LiquityStrategy is
         bytes calldata _lqtySwapData,
         bytes calldata _ethSwapData
     ) public virtual onlyKeeper {
-        if (_swapTarget == address(0))
-            revert StrategySwapTargetCannotBe0Address();
+        _checkSwapTargetForZeroAddress(_swapTarget);
```

Comment:
This is not required here since if it is the zero address it will fail in the next line anyway.

Reply:
Hmm, I see the point. However, we will get the error that swap target is not allowed, which I think is a bit misleading in this case. Wdyt, should we leave it or not? I assume the gas cost for checking if the address is 0 isn't significant, since this will not be called very frequently

Reply:
yes good point. If the gas cost is not much then keeping it makes sense.